### DISCUSSION TITLE
23.11.03 데일리 스크럼

### DISCUSSION THREAD

**Discussion by minju412** [petit-a-petit/MIML#54] [Daily Scrum]
.

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 했나요?
  - 발표(java에서 비동기 다루기)
- 내일은 어떤 일을 하나요?
  - 아티스트 엔티티 생성
  - 아티스트 좋아요 기능

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - 발표 준비
    - 나와 친구인 회원 조회 기능
- 내일은 어떤 일을 하나요?
    - 포스트맨 에러 해결 (feat. 자현님🙇‍♀️)
    - 친구 수락 기능
    - 서브모듈 제거
    - 테스트 코드 작성

**Comment by Nine-JH** [comment]
* 오늘은 어떤 일을 했나요?
  * 못했어요...
* 내일은 어떤 일을 하나요?
  * Developer 설정 확인
